sql/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/ClientE2ETestSuite.scala

```diff
@@ -1670,6 +1670,12 @@ class ClientE2ETestSuite
     }
     checkAnswer(df, (0 until 6).map(i => Row(i)))
   }
+
+  test("SPARK-52770: Support Time type in connect") {
```

Comment:
The test seems good, but the test name seems a bit overkill; "Time type" is OK enough.